src/go/k8s/helm-chart/charts/redpanda-operator/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+name: redpanda
+description: Redpanda operator chart
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+appVersion: 0.1.0
```

Comment:
so what number should we set here? 🤔  there's no 0.1.0 of the operator right now

Reply:
I needed to put something. I can change app version to whatever team would like to.

Reply:
can we use the same as the redpanda version 21.3.4

Reply:
Sure will update.

Reply:
I changed it.